docs/source/model_design/concept_model/results_processing_steps/index.rst

```diff
@@ -24,15 +24,105 @@
 .. _vivarium_best_practices_results_processing:
 
 =========================================================
-Vivarium Results Processing
+Vivarium V&V and Results Processing Tips
 =========================================================
 
 .. contents::
    :local:
    :depth: 1
 
+These are some non-exhaustic tips and tricks!
+
+General things to check/keep in mind
+-------------------------------------
+
+- **Confirm that there are no failed jobs.** The software engineers should check this, but it can be a source of trouble if it is not caught. If there are draw/seed combinations that completed for some scenarios and not others, then the common random numbers will not be enforced across scenarios and results will not be as accurate as expected (outcomes may not exactly match across scenarios when they are expected to).
+
+- **Confirm that you understand what the outputs (and artifact values) represent.** These sorts of misunderstandings can sometimes be the cause of researcher-suspected V&V issues, when really the outputs may meet the V&V targets but they are being interpreted incorrectly. The software engineers are the best resources here! Remember that artifact values generally represent suspectible population incidence rates while GBD values represent total population incidence rates.
+
+- Generally, **all calculations should be done at the draw-specific level** (including calculations of the *differences* between scenarios). Then, once the desired output is calculated at the draw-specific level, take the mean, 2.5th percentile, and 97.5th percentile to generate an overall mean estimate and 95% uncertainty interval. Note that in some situations median values may be desired over means and other uncertainty intervals (such as 90% or 80%) may be desired over 95%.
+
+- Remember that simulation input data are generally age and sex specific, so it's generally easiest to perform verification of simulation inputs and outputs at the age and sex specific level. However, especially if simulating across many years (which may result in slight changes in the population structure over time), be sure to also **evaluate total population outcome rates as well as age- and sex-specific rates**.
+
+- If something is not verifying, but is close to the target, it could be due to stochastic variation. Check and see if it gets closer when you pool across all scenarios, all ages, etc. If so, stoachstic variation could be to blame for the deviation from the target value. In this case, consider increasing the simulated population size and/or simulated number of draws. Also, try looking at the median value, which may be closer to the target value than the mean is when there are small counts.
+
```

Comment:
I know I encountered having untracked population - that might be a helpful starting check to add